robottelo/constants/__init__.py

```diff
@@ -23,6 +23,7 @@
     'zh_TW',
 )
 
+DISTRO_RHEL5 = "rhel5"
```

Comment:
why are we adding rhel5?

Reply:
Because I needed large repos to use it here:
https://github.com/SatelliteQE/robottelo/pull/9331/files#diff-9a55ae74469aa91ad5efb2b9068f73174467cd0c00d45abf7d35f9f2277a7632R697-R708

That list of repos is used for `test_positive_bulk_cancel_sync`, which needs 10 repositories that take a while to sync. I had a lot of trouble debugging that test since small RH repos would finish syncing before I could cancel them.